textgrad/engine/openai.py

```diff
@@ -17,6 +17,7 @@
 from .base import EngineLM, CachedEngine
 from .engine_utils import get_image_type_from_bytes
 
+OLLAMA_BASE_URL = 'http://localhost:11434/v1'
```

Comment:
would it make sense to have this as an env variable? not sure what ppl do with ollama but I guess some might want to change this?